api/apps/api/test/access-control/project-acl/project-acl-get.e2e-spec.ts

```diff
@@ -57,3 +57,14 @@ test(`getting project users as owner with a wrong search query`, async () => {
   );
   fixtures.ThenNoUserInformationIsReturned(response);
 });
+
+test(`getting project users should only show those who are not deleted`, async () => {
+  const projectId = await fixtures.GivenProjectWasCreated();
+  await fixtures.GivenContributorWasAddedToProject(projectId);
+  await fixtures.GivenUserWasAddedToProject(projectId);
+  let response = await fixtures.WhenGettingProjectUsersAsOwner(projectId);
+  fixtures.ThenAllUsersBeforeDeletingAnyFromAppAreReturned(response);
+  await fixtures.GivenUserIsDeleted();
+  response = await fixtures.WhenGettingProjectUsersAsOwner(projectId);
+  fixtures.ThenAllUsersExceptDeletedAreReturned(response);
```

Comment:
minor, maybe we could split this test into two - one that tests the first `Then...` assertion in the current code, one that doesn't assert that but goes one deleting the user and asserting that `ThenAllUsersExceptDeletedAreReturned`.

Reply:
The first `Then` is actually there just as a check of the current users added to the project as expected for this test (1 owner, 1 contributor and 1 new added user). There is actually already a test to check for the regular `Getting Project Users As Owner` a little bit above this code.

In this case I specified it because I was creating a new user because of the principle of keeping the seeds as they are when starting the application (because I have to delete a user and deleting one of the 'aa' 'bb' etc users can interfere in other tests).

So long story short: first `Then` is already tested in a different way elsewhere and the truly important part is the last `Then` here.